### ISSUE TITLE
Store executable compilation/configuration options, if possible

### ISSUE THREAD

**Issue by apdavison** [fipymigrate/sumatra#16]
_Imported from Bitbucket issue [15](https://bitbucket.org/apdavison/sumatra/issue/15),  created by apdavison on 06-26-2010 at 21:17, last modified: 04-04-2013 at 09:40_

**Comment by apdavison** [comment]
I do not think this is possible in the general case, at least not without major reverse-engineering skills.

.NET executables apparently store a lot of metadata, so that could be useful, if anyone is interested in running Sumatra using IronPython.

For Python, there is a lot of information available in distutils.sysconfig. I guess other programs might have their own equivalents.

For systems using package management, there might be information available.

It might be possible to allow people who have compiled from source to register their configure script/Makefile, etc. with Sumatra.

In summary, this is a big job.

        

_Trac comment by apdavison on 07-20-2010 at 08:00_
